### PR TITLE
support for snapshots (to be used as backups)

### PR DESCRIPTION
**Pull request by cesargms**

### CODE REVIEW

File: lib/python/treadmill_aws/cli/admin/aws/image.py [hunk 1]
```diff
@@ -168,6 +168,46 @@ def create(base_image, base_image_account, userdata, instance_profile,
         )
         click.echo(instance['Instances'][0]['InstanceId'])
 
+    @image.command()
```
Comment (andreikeis) [inline lib/python/treadmill_aws/cli/admin/aws/image.py]: image.command(name="create-from-snapshot")

File: lib/python/treadmill_aws/cli/admin/aws/instance.py [hunk 2]
```diff
@@ -129,16 +129,30 @@ def configure(instance):
...
+    )
     @treadmill_aws.cli.admin.aws.ON_AWS_EXCEPTIONS
     def create(
             image, image_account, count, disk_size, key, role, ip_address, eni,
             spot, secgroup, size, subnet, data, instance_profile,
-            hostgroup, hostname):
+            hostgroup, hostname, domain, ipa_enroll):
         """Create instance(s)"""
-        ipa_client = awscontext.GLOBAL.ipaclient
+        if ipa_enroll:
```
Comment (andreikeis) [inline lib/python/treadmill_aws/cli/admin/aws/instance.py]: i do not see where ipa_client being used?

Comment (cesargms) [reply lib/python/treadmill_aws/cli/admin/aws/instance.py]: further down in call hostmanager.host_create()

File: lib/python/treadmill_aws/cli/admin/aws/image.py (hunk 1 above)
Comment (cesargms) [reply lib/python/treadmill_aws/cli/admin/aws/image.py]: fixed.

**Pull request merged**